v3/pkg/application/menuitem_roles.go

```diff
@@ -47,7 +47,12 @@ func NewCutMenuItem() *MenuItem {
 		SetAccelerator("CmdOrCtrl+x").OnClick(func(ctx *Context) {
 		currentWindow := globalApplication.CurrentWindow()
 		if currentWindow != nil {
```

Comment:
We should add `cut()` to the `Window` interface. It can be done in a different PR